flower_reader.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+FILENAME = 'flowers.txt'
+user_name = input('Enter your first and last name: ')
+find_first_letters_of_user_name = lambda user_name: user_name[0].title()
+
+def create_flower_dictionary(file_name):
+    """Read in a file and create a dictionary"""
+
+    flowers = {}
+
+    with open(file_name) as f_object:
+        contents = f_object.readlines()
+        for flower in contents:
+            split_flower_content = flower.split(" ", 1)
+            letter = split_flower_content[0]
```

Comment:
From the looks of this code, it appears to split on the space and the first split would then contain the colon, such as `A:` and the second the flower name: `African Daisy`. Thus, I'm thinking you want to split with `flower.split(": ", 1)`. But I might be wrong. And if this is the case, you wouldn't have to use the `maxsplit` argument of `1`. (although I learned something there!)

Reply:
@paulghaddad It does not appear that changing the split from ":" to " " affects anything. So I just kept it as it was. Does that work?

Reply:
👍